specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/security.json

```diff
@@ -727,6 +727,116 @@
+          "get": {
+            "x-ms-examples": {
+              "Gets the Advanced Threat Protection settings for the specified resource.": {
+                "$ref": "./examples/AdvancedThreatProtection/GetAdvancedThreatProtectionSettings_example.json"
+              }
+            },
+            "tags": [ "AdvancedThreatProtection" ],
+            "description": "Gets the Advanced Threat Protection settings for the specified resource.",
+            "operationId": "AdvancedThreatProtection_Get",
+            "parameters": [
+              {
+                "$ref": "#/parameters/ApiVersion"
+              },
+              {
+                "$ref": "#/parameters/ResourceUri"
+              },
+              {
+                "$ref": "#/parameters/AdvancedThreatProtectionSettingName"
+              }
+            ],
+            "responses": {
+              "200": {
+                "description": "Successful request to get Advanced Threat Protection settings.",
+                "schema": {
+                  "$ref": "#/definitions/AdvancedThreatProtectionSettings"
+                }
+              },
+              "default": {
+                "description": "Error response describing why the operation failed.",
+                "schema": {
+                  "$ref": "#/definitions/CloudError"
+                }
+              }
+            }
+          },
+          "put": {
+            "x-ms-examples": {
+              "Creates or updates the Advanced Threat Protection settings on a specified resource.": {
+                "$ref": "./examples/AdvancedThreatProtection/PutAdvancedThreatProtectionSettings_example.json"
+              }
+            },
+            "tags": [ "AdvancedThreatProtection" ],
+            "description": "Creates or updates the Advanced Threat Protection settings on a specified resource.",
+            "operationId": "AdvancedThreatProtection_Put",
+            "parameters": [
+              {
+                "$ref": "#/parameters/ApiVersion"
+              },
+              {
+                "$ref": "#/parameters/ResourceUri"
+              },
+              {
+                "$ref": "#/parameters/AdvancedThreatProtectionSettingName"
+              },
+              {
+                "$ref": "#/parameters/AdvancedThreatProtectionSettings"
+              }
+            ],
+            "responses": {
+              "200": {
+                "description": "Successful request to put Advanced Threat Protection settings.",
+                "schema": {
+                  "$ref": "#/definitions/AdvancedThreatProtectionSettings"
+                }
+              },
+              "default": {
+                "description": "Error response describing why the operation failed.",
+                "schema": {
+                  "$ref": "#/definitions/CloudError"
+                }
+              }
+            }
+          },
+          "delete": {

```

Comment:
delete doesn't have a lot of meaning here
what's the difference between this and "isEnabled": false?

please pick one of them (is enabled property or create delete a resource)
I suggest the property since it will allow you to use policies

Reply:
make sense, I removed the DELETE operation